tfx/components/statistics_gen/executor.py

```diff
@@ -107,10 +109,12 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
         stats_options.schema = schema
 
     split_uris = []
+    exclude_splits = exec_properties[EXCLUDE_SPLITS_KEY]
     for artifact in input_dict[EXAMPLES_KEY]:
       for split in artifact_utils.decode_split_names(artifact.split_names):
-        uri = os.path.join(artifact.uri, split)
-        split_uris.append((split, uri))
+        if not exclude_splits or split not in exclude_splits:
+          uri = os.path.join(artifact.uri, split)
```

Comment:
let's use the artifact_utils.get_split_uri(xxx, split) (this should check if split_name is set correctly)